### ISSUE TITLE
Qudi modifies system path variable

### ISSUE THREAD

**Issue by Geegee** [Ulm-IQO/qudi-core#12]
### Version

v1.0.1

### What is affected by the bug?

I tried using different versions of Pyside2 5.15.2.1 / 5.15.2 / 5.15.1 to no avail.
Just running pyside2-uic "some-gui-module.ui" runs without error.
As @kay-jahnke pointed out it seems like the environment qudi runs in 
does not have access to "pyside2-uic". 
Yet in the same terminal qudi is started from and in the same environment  pyside2-uic can 
be run from the command line. 

This shows the output when run sys.path within qudi:
![image](https://user-images.githubusercontent.com/11192737/152781163-3bb00f6d-09d9-4776-b9b3-a253cc47327d.png)
The output where pyside2-uic resides
![image](https://user-images.githubusercontent.com/11192737/152781253-acd78c65-5de3-417a-90b3-11b7d4a68412.png)
This is when run in the ipython console
![image](https://user-images.githubusercontent.com/11192737/152781577-3e4acbdc-d561-4985-8ff9-ba818c729b40.png)
It seems like qudi is modifying the path variable, which I am not sure how to fix at the moment.

### When does the bug occur?

Trying to load Gui modules with a "*.ui" file.

### How do we replicate the issue?

Seems to occur only on Linux.

### Expected behavior

Create a uic file.

### Relevant log output
Exception during activation:
```
FileNotFoundError: [Errno 2] No such file or directory: 'pyside2-uic "/tmp/tmp57zk2x42"'
Traceback (most recent call last):

  File "/home/geegee/Documents/Programme/qudi-core/src/qudi/core/module.py", line 286, in __activation_callback
    self.on_activate()

  File "/home/geegee/Documents/Programme/qudi-iqo-modules/src/qudi/gui/time_series/time_series_gui.py", line 115, in on_activate
    self._mw = TimeSeriesMainWindow()

  File "/home/geegee/Documents/Programme/qudi-iqo-modules/src/qudi/gui/time_series/time_series_gui.py", line 45, in __init__
    loadUi(ui_file, self)

  File "/home/geegee/Documents/Programme/qudi-core/src/qudi/util/uic.py", line 70, in loadUi
    result = subprocess.run(f'pyside2-uic "{file_path}"',

  File "/usr/lib/python3.8/subprocess.py", line 489, in run
    with Popen(*popenargs, **kwargs) as process:

```

### Contact Details

wgolff@googlemail.com

**Comment by drogenlied** [comment]
You did not show the actual error here, can you post that?

**Comment by Neverhorst** [comment]
The only place where the path could differ from the qudi process is [here](https://github.com/Ulm-IQO/qudi-core/blob/main/src/qudi/util/uic.py#L70-L73)
I think `subprocess.run` can be configured accordingly. That's the place where I would start searching for the bug.

**Comment by Geegee** [comment]
Thanks for pointing it out. I put the error message under the 'log 
output' section now, sorry.

**Comment by Neverhorst** [comment]
Has been resolved by merging PR #18.
